### ISSUE TITLE
wrong line number type /deprecated

### ISSUE THREAD

**Issue by Jan-Zimmermann** [jjchiw/gelf4net#43]
Hi,

Gelf4Net sends field line number as string. The GELF standard describes it as number. Unfurtunatly log4net seems to fill sometimes the internal variable with an '?'. This end up with an index Error on Graylog Server.

additionally the field line,file and facility is marked as deprecated.

**Comment by jjchiw** [comment]
This error I think it only happens when you use an Async~Appender even is not specify the workaround is this https://github.com/jjchiw/gelf4net#async-appenders

```
Async~Appenders doesn't log the file an line and IncludeLocationInformation should be set as false (default) <param name="IncludeLocationInformation" value="false"/>
```

**Comment by Jan-Zimmermann** [comment]
Yes, found this workaround too. And collecting location Information is slow too. But you should consider to change Type of line Property to integer, so this will not ever happen.

**Comment by jjchiw** [comment]
Hi, Fixed in the new packages 3.0.0.2 and *Appenders 1.0.0.6, but if you're using donetcore the location information is not working
